src/pytorch_lightning/callbacks/progress/rich_progress.py

```diff
@@ -451,13 +451,12 @@ def on_predict_batch_end(self, trainer, pl_module, outputs, batch, batch_idx, da
 
     def _get_train_description(self, current_epoch: int) -> str:
         train_description = f"Epoch {current_epoch}"
+        if self.trainer.max_epochs is not None:
+            train_description += f"/{self.trainer.max_epochs - 1}"
```

Comment:
what if it's -1?

Reply:
If not, it will start from 0/24, and end with 23/24